docs/services/frontend.md

```diff
@@ -0,0 +1,174 @@
+# frontend
+
+The frontend is responsible to provide a UI for users, as well
+as an API leveraged by the UI or other clients. The application is based on
+[Next.JS](https://nextjs.org/) to provide a React web-based UI and API routes.
+
+[Frontend source](../../src/frontend/)
+
+## SDK initialization
+
+It is recommended to use a Node required module when starting your NodeJS
+application to initialize the SDK and auto-instrumentation. When initializing
+the OpenTelemetry NodeJS SDK, you optionally specify which auto-instrumentation
+libraries to leverage, or make use of the `getNodeAutoInstrumentations()`
```

Comment:
Two options are described. It could be added that which one we selected for the demo app and why.